instrumentation/active_storage/lib/opentelemetry/instrumentation/active_storage/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module ActiveStorage
+      VERSION = '0.1.0'
```

Comment:
@open-telemetry/ruby-contrib-approvers Our automated release tooling handles versioning. For initial releases this should have been set to `0.0.0`

The instrumentation generator seeds the version here:

https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/.instrumentation_generator/templates/lib/instrumentation/instrumentation_name/version.rb.tt#L10 

That along with the empty CHANGELOG signals to the release tooling that this specific version will need to be an "initial release" of the gem https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/.instrumentation_generator/templates/CHANGELOG.md.tt

Reply:
Thanks for pointing this out, Ariel. I'm sorry about missing it before the merge.